### DISCUSSION TITLE
Activity under House Information

### DISCUSSION THREAD

**Discussion by maccident** [dwainscheeren/dwains-lovelace-dashboard#374] [Q&A]
I love this dashboard.  It has given me what I've always wanted in a really convenient package.

One question I have is, how do I get the activity section to populate under House Information?
I'd love to see my alarm system zones trigger when someone uses a door, or when my exterior motion sensors trigger - but so far, it just remains blank all the time.

Any ideas?  I can't find anything in the documentation that mentions Activity.

**Comment by loca5790** [comment]
having same issue been trying to figure out the same thing for a while.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
